src/components/Navbar.tsx

```diff
@@ -45,168 +44,191 @@ export const NavbarHtml = async ({ session, activePage }: Props) => {
   const routes = pageRoutes.filter(
     (x) => !x.roles || x.roles?.every((x) => userRoles.includes(x)),
   );
-
-  const toggleMobileMenuHyperscript = `on click toggle [@data-open] on #mobile-menu-button then toggle between .block and .hidden on #mobile-menu`;
-  const toggleUserMenuHyperscript = `on click toggle between .block and .hidden on #user-menu`;
-
   return (
-    <nav>
-      <div class="mx-auto px-2 sm:px-6 lg:px-8">
-        <div class="relative flex h-16 items-center justify-between">
-          <div class="absolute inset-y-0 left-0 flex items-center lg:hidden">
-            <button
-              type="button"
-              id="mobile-menu-button"
-              class="group relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
-              _={toggleMobileMenuHyperscript}
-            >
-              <span class="absolute -inset-0.5"></span>
-              <svg
-                id="mobile-menu-open"
-                class="block h-6 w-6 group-data-[open]:hidden"
-                fill="none"
-                viewBox="0 0 24 24"
-                stroke-width="1.5"
-                stroke="currentColor"
-              >
-                <path
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
-                />
-              </svg>
-              <svg
-                id="mobile-menu-close"
-                class="hidden h-6 w-6 group-data-[open]:block"
-                fill="none"
-                viewBox="0 0 24 24"
-                stroke-width="1.5"
-                stroke="currentColor"
+    <>
+      <script>
+        {function toggleMobileMenu() {
```

Comment:
Hmm, looks like you've brought back the inline javascript and removed hyperscript :P